src/main/java/io/github/moulberry/notenoughupdates/profileviewer/MiningPage.java

```diff
@@ -434,7 +439,12 @@ public void drawPage(int mouseX, int mouseY, float partialTicks) {
 						"",
 						"§7Increases the chance for you to",
 						"§7trigger rare occurrences in",
-						"§2Dwarven Mines " + EnumChatFormatting.GRAY + "by " + EnumChatFormatting.GREEN + luckOfCave + "%§7."
+						"§2Dwarven Mines " + EnumChatFormatting.GRAY + "by " + EnumChatFormatting.GREEN + luckOfCave + "%§7.",
+						"",
+						EnumChatFormatting.GRAY  + "Rare occurrences include:",
+						EnumChatFormatting.GRAY  + "• " + EnumChatFormatting.GOLD + "Golden Goblins",
+						EnumChatFormatting.GRAY  + "• " + EnumChatFormatting.DARK_PURPLE + "Fallen Stars",
+						EnumChatFormatting.GRAY  + "• " + EnumChatFormatting.GOLD + "Powder Ghasts"
```

Comment:
In the real lore there is a space before the •

Reply:
alright